binrw_derive/src/parser/meta_types.rs

```diff
@@ -232,10 +271,10 @@ mod tests {
         );
     }
 
-    try_parse!(ident_pat_type, IdentPatType, { foo: u8 });
-    try_parse_fail!(ident_pat_type_missing_ident, IdentPatType, { : 3u8 });
-    try_parse_fail!(ident_pat_type_missing_ty, IdentPatType, { foo: });
-    try_parse_fail!(ident_pat_type_wrong_ty_type, IdentPatType, { foo: 3u8 });
+    try_parse!(ident_pat_type, NamedImport, { foo: u8 });
+    try_parse_fail!(ident_pat_type_missing_ident, NamedImport, { : 3u8 });
+    try_parse_fail!(ident_pat_type_missing_ty, NamedImport, { foo: });
+    try_parse_fail!(ident_pat_type_wrong_ty_type, NamedImport, { foo: 3u8 });
```

Comment:
tests should probably be renamed to have something to do with NamedImport, should add some cases dealing with the `=` now optionally included, and add some cases for NamedArg.